### PR TITLE
Add more test to cover arch.py

### LINKED ISSUE #420: Add more unittest coverage for src in bundle-workflows/src/system
**Is your feature request related to a problem? Please describe.**
Currently we dont have a full test coverage for bundle-workflows/src/system

```
---------- coverage: platform darwin, python 3.7.7-final-0 -----------
Name                                Stmts   Miss  Cover
-------------------------------------------------------
src/system/__init__.py                  0      0   100%
src/system/arch.py                      8      0   100%
src/system/console.py                   4      4     0%
src/system/execute.py                   8      8     0%
src/system/properties_file.py          38      1    97%
src/system/temporary_directory.py      12      0   100%
src/system/working_directory.py         7      0   100%
-------------------------------------------------------
TOTAL                                  77     13    83%
```

We need to add more tests to the code.

**Describe the solution you'd like**
Add more test coverage for src in bundle-workflows

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by peterzhuamazon**
Signed-off-by: Peter Zhu <zhujiaxi@amazon.com>

### Description
Add more test to cover arch.py
 
### Issues Resolved
#420
 
### Check List
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: bundle-workflow/tests/tests_system/test_arch.py [hunk 1]
```diff
@@ -15,6 +15,18 @@ class TestArch(unittest.TestCase):
     def test_current_arch(self):
         self.assertTrue(current_arch() in ["x64", "arm64"])
 
+    @patch("subprocess.check_output")
```
Comment (dblock) [inline bundle-workflow/tests/tests_system/test_arch.py:18]: I think you can do `@patch("subprocess.check_output", return_value=...)` which would make things a bit cleaner.

File: bundle-workflow/tests/tests_system/test_arch.py [hunk 2]
```diff
@@ -15,6 +15,18 @@ class TestArch(unittest.TestCase):
...
+    def test_x64_arch(self, mock_subprocess):
+        mock_subprocess.return_value = "x86_64".encode()
+        self.assertTrue(current_arch() == "x64")
+
+    @patch("subprocess.check_output")
+    def test_arm64_arch(self, mock_subprocess):
```
Comment (dblock) [inline bundle-workflow/tests/tests_system/test_arch.py:24]: Split into 2 tests.

**Review by dblock — APPROVED** [review]
I am nitpicking, feel free to make these suggestions before/after merge.

File: bundle-workflow/tests/tests_system/test_arch.py (hunk 1 above)
Comment (peterzhuamazon) [reply bundle-workflow/tests/tests_system/test_arch.py:18]: I will address this in another improvement PR.

File: bundle-workflow/tests/tests_system/test_arch.py (hunk 2 above)
Comment (peterzhuamazon) [reply bundle-workflow/tests/tests_system/test_arch.py:24]: I will address this in another improvement PR.

**Review by mch2 — APPROVED** [review]

**Pull request merged**